lib/mp4/transmuxer.js

```diff
@@ -978,20 +987,23 @@ clearDtsInfo = function(track) {
  * DTS the transmuxer has ever seen and the minimum DTS for the
  * current track
  */
-calculateTrackBaseMediaDecodeTime = function(track) {
+calculateTrackBaseMediaDecodeTime = function(track, keepOriginalTimestamps) {
```

Comment:
Document params here, too?  I know track docs were already missing, but it wouldn't hurt to add them here for both params.